src/lib/methods/batch/batch-write.ts

```diff
@@ -0,0 +1,86 @@
+import { Executable } from '../executable';
+import { DynamoDB } from '../../dynamodb';
+import { BatchItem } from './batch-item';
+import { BatchMethod } from './batch-method';
+import { BatchWriteItemInput } from 'aws-sdk/clients/dynamodb';
+import pRetry from 'p-retry';
+
+export class BatchWrite extends BatchMethod implements Executable {
+
+	constructor(
+		dynamodb: DynamoDB,
+		private items: BatchItem[],
+		private retryCount = 5,
+		private randomizeRetry = true) {
+		super(dynamodb);
+	}
```

Comment:
```suggestion
	constructor(
		dynamodb: DynamoDB,
		private items: BatchItem[],
	) {
		super(dynamodb);
	}
```

As explained below, the retry configuration should be set with the `.retry()` method.